README.md

```diff
@@ -19,7 +19,8 @@ Before using any of our Docker images, you'll need to create a .env file. The re
 
 * db: The MySQL database storing NYC city data both in semi-raw form and joined into a model for the Rails website. The database can be examined with `docker exec deltanycheatseek_db_1 mysql -u <user from .env> -p <password from .env>`. Local clients can connect at 127.0.0.1:3306 ([localhost will not work](http://stackoverflow.com/a/32361238/103315)).
 * db-udf: Installs [user-defined functions for making regular expression queries](https://github.com/mysqludf/lib_mysqludf_preg) required for data-cleaning into MySQL. Only needs to be run once.
-* nb: Installs Python and starts a Jupyter notebook at [localhost:8888](http://localhost:8888). Can also be used to run the data-import scripts: run `docker exec -it deltanycheatseek_nb_1 /bin/bash`, then execute the scripts
+* luigi: Runs all import scripts using [Luigi](http://luigi.readthedocs.io). Will skip imports already run by the container. Imports can be run in parallel by adjusting the LUIGI_WORKERS parameter in your .env file.
+* nb: Installs Python and starts a Jupyter notebook at [localhost:8888](http://localhost:8888).
```

Comment:
I think we need one more note here that both the `luigi` and `nb` containers are for running imports, just in a different way. Generally, separate services have separate dependencies, code etc so seeing two services with the same requirements and code threw me a bit.

Next time we meet, should get a sense of who might still be using the nb container for notebooks. If no one's really using it, it might be worthwhile to kill it, and rely on the luigi container.